04_environment/index.tex

```diff
@@ -1 +1,400 @@
+\end{center}
+\end{table}
+
+\subsubsection{Tier System}
+
+The deer hunting and fishing is divided into tiers (the number of fish that can be caught in a given day) which depends on the total input resources. Where the “zero” tier represents the cost of getting to the hunting location.\\ 
+
+Beyond the zero tier we have the cost of catching another deer/fish, which decreases as it is easier to catch the second animal than the first since we have already arrived at the hunting location. It should be noted there is a limit to how many animals we can catch in a given day.\\
+
+The focus of fishing is to avoid risk at the expense of payoff. Thus all the tier requirements are lower than the deers however the return is also lower, in addition to this the “zero” tier has been removed. Thus, if only 1 island goes fishing they are still expected to have a low return as long as they have enough resources to catch the first fish. \\
+
+The focus of deer hunting is higher payoffs therefore it has better returns but all of its tier thresholds are significantly higher and it also has a “zero” tier. Therefore it is possible for a single island to go deer hunting without enough resources to reach the hunting location leading to no return.
+
+\begin{equation}
+Increments\ of\ catching\ n\ deer's=
+\left( \begin{array}{ll@{}}
+n=0 -> \Delta^0 = 1 \\
+n=1 -> \Delta^1 = 0.8^1 \\
+n=2 -> \Delta^2 = 0.8^2 = 0.64 \\
+n=2 -> \Delta^3 = 0.8^3 = 0.512 \\
+n=4 -> \Delta^4 = 0.8^4 = 0.4096\\
+\end{array} \right) 
+\end{equation}
+
+\begin{equation}
+Cumulative\ deer \ cost \sum_{n=0}^{n} \Delta^{n} = \Delta^{0} + \Delta^{1} + \Delta^{2} + \Delta^{3} + \Delta^{4} + ....
+\end{equation}
+
+\begin{equation}
+Cumulative\ fishing\ cost \ = \ \sum_{n=1}^{n} \Delta^{n} = \Delta^{1} + \Delta^{2} + \Delta^{3} + \Delta^{4} + \Delta^{5} + ....
+\end{equation}
+
+Equations that show the formulation of the decay function for tier system.\\
+
+The tier system is shown above, where n stands for the tiers, with decay costs ($\Delta$) of 0.8 for both the deer hunt and fishing. Fishing cost starts at n=1 due to the reduced cost.\\
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=0.9\textwidth]{04_environment/Images/Foraging Tier System.PNG}
+    \caption{Foraging Tier System}
+    \label{Images:Foraging Tier System}
+\end{figure}
+
+As the tiers increase the cost per animal decreases but the total cost increases up to a daily catch limit. Thus if teams collaborate, they could invest less as individuals to get a good return through deer hunting, however if they invest too much then they will over spend as they reach the daily max catch. Thus it would be beneficial for some teams to go fishing which has a higher maximum catch per day but comes with lower returns (thus there is a need for some self sacrifice to be done to reach maximum welfare). \\ 
+
+Fishing has similar functions with the key difference being the Tiers start at 1 and Normal distribution. \\
```

Comment:
bad sentence  -- revise.

why capital letter for tiers ??

Reply:
I believe your authors have a tendency to use capital letters for places I don't deem it to be necessary... weird :D

Reply:
I don't think there is an issue here with capitalisation in the sense that if it is a term being used to formally descripe an aspect of the game, it can be capitalised. In the same way, the Game State is capitalised because it is a formal term we use to describe the simulation. Whether you capitalise it or not, I think it is only consistency that matters here.

Reply:
consistently reference the same term the same way - daily max catch == daily catch limit but they're different ways of saying the same thing - it would be clearer to be consistent

Suggested rewrite:

Higher tiers result in a lower cost per animal, however, the total cost increases up to a daily catch limit.
This means that if teams collaborate, they can invest less as individuals to get a collectively greater return through deer hunting. If they invest too much then they will over spend as they reach the daily catch limit. As a result, it is more beneficial for some teams to go fishing because it has a higher daily catch limit, although it comes with lower returns. In other words, there is a need for some self sacrifice to be done to reach maximum welfare.